console/src/components/Login/Login.js

```diff
@@ -1,96 +1,115 @@
-                      className={styles.form_input + ' form-control'}
-                      type="password"
-                      placeholder={`Enter ${globals.adminSecretLabel}`}
-                      name="password"
-                    />
-                  </div>
+  // form submit handler
+  const onSubmit = e => {
+    e.preventDefault();
+    const successCallback = () => {
+      setLoading(false);
+      setError(null);
+    };
+    const errorCallback = err => {
+      setAdminSecretInput('');
+      setLoading(false);
+      setError(err);
+    };
+    setLoading(true);
+    verifyLogin({
+      adminSecret: adminSecretInput,
+      shouldPersist,
+      successCallback,
+      errorCallback,
+      dispatch,
+    });
+  };
+
+  return (
+    <div className={styles.mainWrapper + ' container-fluid'}>
+      <div className={styles.container + ' container'} id="login">
+        <div className={styles.loginCenter}>
+          <Helmet title={'Login | ' + 'Hasura'} />
+          <div className={styles.hasuraLogo}>
+            <img src={hasuraLogo} />
+          </div>
+          <div className={styles.loginWrapper}>
+            <form className="form-horizontal" onSubmit={onSubmit}>
+              <div className={styles.input_addon_group + ' ' + styles.padd_top}>
+                <div className={'input-group ' + styles.input_group}>
+                  <input
+                    onChange={onAdminSecretChange}
+                    className={styles.form_input + ' form-control'}
+                    type="password"
+                    placeholder={`Enter ${globals.adminSecretLabel}`}
+                    name="password"
+                  />
                 </div>
-                <div className={styles.signin_btn}>
-                  <Button type="submit" color="green" className="form-control">
-                    {loginText}
-                  </Button>
+              </div>
+              <div className={styles.signin_btn}>
+                <Button
+                  type="submit"
+                  color="green"
+                  className="form-control"
+                  disabled={loading}
+                >
+                  {loginText}
+                </Button>
+              </div>
+              <div className={`${styles.display_flex} ${styles.add_pad_left}`}>
+                <input
+                  type="checkbox"
+                  checked={shouldPersist}
+                  onChange={toggleShouldPersist}
+                  className={`${styles.add_mar_right_small} ${
+                    styles.remove_margin_top
+                  }`}
+                />
+                <label className={styles.add_mar_right_mid}>Remember in this browser</label>
+                <div style={{ marginTop: '2px' }} className={styles.cursorPointer}>
+                  <i className={'fa fa-question-circle'} data-toggle="tooltip" data-placement="bottom" title="Do not check this in production" />
```

Comment:
Tooltip not required here. This message raises questions to why should I not check it in production. Users are smart enough to understand what 'remember me' means and its risks.